inspirehep/modules/authors/views.py

```diff
@@ -225,26 +227,35 @@ def new():
     return render_template('authors/forms/new_form.html', form=form, **ctx)
 
 
+def _author_record_from_db(recid):
+    """Get author record from DB or return None if it is not there."""
+    try:
+        return get_db_record('aut', recid)
+    except RecordGetterError:
+        return None
+
+
+def _author_record_from_legacy(recid):
+    """Get author record from Legacy or return None if it is not there."""
+    try:
+        url = get_legacy_url_for_recid(recid) + '/export/xm'
+        xml_response = requests.get(url)
+        xml = fromstring(xml_response.content)
+        xml_record = xml.xpath("//*[local-name() = 'record']")[0]
```

Comment:
why this strange selector and not simply `//record`?

Reply:
"Since there is no namespace configured in the XPath expression, we don’t find any matching nodes. The XPath Expression is looking for default namespace but the document is with a different namespace. To resolve this problem of namespaces we have to use the local-name() function. The local-name() function ignores the namespace and returns the query results as if the XML did not have any namespace." Source: https://examples.javacodegeeks.com/core-java/xml/xpath/xpath-local-name-example/